src/components/UI/Card/Card.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import "./Card.css";
+import CardBody from "./CardBody/CardBody";
+import CardHeader from "./CardHeader/CardHeader";
+import classNames from "classnames";
+
+function Card(props) {
+  const [cbValue, setCbValue] = useState(""); //checkbox state
+  const [penClicked, setPenClicked] = useState(false); //true - card body is content editable
+  //current header and body values
+  const [currentValue, setCurrentValue] = useState(props.CardText);
+  //last saved header and body values to cancel changes
+  const [lastValue, setLastValue] = useState(props.CardText);
+
+  //dark styles mode if checkbox is checked
+  const cardClasses = classNames("card", { "dark-mode-card": cbValue });
+
+  const cbValueHandler = (val) => {
+    setCbValue(val); //setting checkbox state
+  };
+
+  const penClickedHandler = (val) => {
+    setPenClicked(val); //setting/unsetting edit mode
+  };
```

Comment:
The same is actually here. Bool type - you can set !penClicked.

Also, naming should be meaningful to what your object actually do for a component it belongs to. Ex.: Clicking pencil icon is used to move card to Edit Mode. So more meaningful could be to use state name 'editMode' or 'isEdited' or smth like that.